src/apps/stable/routes/session/forgotPassword/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useCallback, useState } from 'react';
+import Page from 'components/Page';
+import { useNavigate } from 'react-router-dom';
+import globalize from 'lib/globalize';
+import Button from 'elements/emby-button/Button';
+import Input from 'elements/emby-input/Input';
+import { useMutation } from '@tanstack/react-query';
+import Dashboard from 'utils/dashboard';
+import { getUserApi } from '@jellyfin/sdk/lib/utils/api/user-api';
+import { ForgotPasswordAction } from '@jellyfin/sdk/lib/generated-client/models/forgot-password-action';
+import ServerConnections from 'components/ServerConnections';
+
+export const ForgotPasswordPage = () => {
+    const navigate = useNavigate();
+    const [username, setUsername] = useState('');
+
+    const forgotPasswordMutation = useMutation({
+        mutationFn: async (enteredUsername: string) => {
+            const currentApi = ServerConnections.getCurrentApi();
+            if (!currentApi) {
+                throw new Error('API not available');
+            }
+            const response = await getUserApi(currentApi).forgotPassword({
+                forgotPasswordDto: {
+                    EnteredUsername: enteredUsername
+                }
+            });
+
+            return response.data;
+        },
+        onSuccess: (result) => {
+            if (result.Action == ForgotPasswordAction.ContactAdmin) {
+                Dashboard.alert({
+                    message: globalize.translate('MessageContactAdminToResetPassword'),
+                    title: globalize.translate('ButtonForgotPassword')
+                });
+            }
+
+            if (result.Action == ForgotPasswordAction.InNetworkRequired) {
+                Dashboard.alert({
+                    message: globalize.translate('MessageForgotPasswordInNetworkRequired'),
+                    title: globalize.translate('ButtonForgotPassword')
+                });
+            }
+
+            if (result.Action == ForgotPasswordAction.PinCode) {
+                let msg = globalize.translate('MessageForgotPasswordFileCreated');
+                msg += '<br/><br/>';
+                msg += globalize.translate('MessageForgotPasswordPinReset');
+                msg += '<br/><br/>';
+                msg += result.PinFile;
+                msg += '<br/>';
+                Dashboard.alert({
+                    message: msg,
+                    title: globalize.translate('ButtonForgotPassword'),
+                    callback: function () {
+                        navigate('forgotpasswordpin');
```

Comment:
```suggestion
                        navigate('/forgotpasswordpin');
```
When I click submit I get navigated to a not found page. Possible fix? (haven't tried)

Reply:
Yes, i have tried and it works, ty